Core/ELF/ParamSFO.cpp

```diff
@@ -276,3 +280,34 @@ void ParamSFOData::ValueData::SetData(const u8* data, int size)
 	u_size = size;
 }
 
+std::string ParamSFOData::generateFakeID(std::string filename)
```

Comment:
Sorry about capital letter, will correct, I knew about curly, but this whole file uses it on newline so I just based it on what I saw.